partiql-eval/src/main/kotlin/org/partiql/eval/internal/Compiler.kt

```diff
@@ -40,73 +41,74 @@ import org.partiql.plan.Statement
 import org.partiql.plan.debug.PlanPrinter
 import org.partiql.plan.visitor.PlanBaseVisitor
 import org.partiql.spi.fn.FnExperimental
+import org.partiql.types.StaticType
 import org.partiql.value.PartiQLValueExperimental
 import org.partiql.value.PartiQLValueType
 import java.lang.IllegalStateException
 
 internal class Compiler(
     private val plan: PartiQLPlan,
-    private val session: PartiQLEngine.Session
-) : PlanBaseVisitor<Operator, Symbols>() {
+    private val session: PartiQLEngine.Session,
+    private val symbols: Symbols
+) : PlanBaseVisitor<Operator, StaticType?>() {
```

Comment:
(Following rebase of partiql-eval) Change `ctx` to have type `StaticType?`. Initialize `symbols` as part of `Compiler` constructor since it is not scope sensitive.